plugins/kiali-backend/src/clients/fetch.ts

```diff
@@ -147,7 +147,8 @@ export class KialiFetcher {
       };
     }
 
-    requestInit.url = new URL(endpoint, this.KialiDetails.url).href;
+    const loginUrl = `${this.KialiDetails.url}/${endpoint}`;
```

Comment:
Toa void results like : **https://kiali-istio-system.apps-crc.testing/kiali//api/status** with 2 "/" I would add

```ts
const loginUrl = `${this.KialiDetails.url}/${endpoint.replace(/^\//g, "");}`;
```

To be sure that it's going to work. What do you think?

Reply:
Thanks for the suggestion, @aljesusg ! 
I have included for testing: 
`url: http://localhost:20001/kiali/`
And works fine.

Reply:
> Toa void results like : **https://kiali-istio-system.apps-crc.testing/kiali//api/status** with 2 "/" I would add
> 
> ```ts
> const loginUrl = `${this.KialiDetails.url}/${endpoint.replace(/^\//g, "");}`;
> ```
> 
> To be sure that it's going to work. What do you think?

MMM the same way we should be sure that user is going to set *https://kiali-istio-system.apps-crc.testing/kiali* and not  *https://kiali-istio-system.apps-crc.testing/kiali/*  (Without the last */*) this is why I wanted to use the URL class

```ts
new URL("/api/status", "https://kiali-istio-system.apps-crc.testing/").href
'https://kiali-istio-system.apps-crc.testing/api/status'
```

But I don't know how to deal with a pathname with this class ....maybe
```ts
const baseUrl = new URL(this.KialiDetails.url);
const pathName = baseUrl.pathname
const host = baseUrl.href.replace(pathName,"")
const target = new URL(`${pathName}${endpoint}`, host).href
```